static/js/chart/draw.ts

```diff
@@ -469,37 +469,49 @@ function drawLineChart(
   }
 }
 
-interface PlotParams {
-  // Label to color.
-  colors: { [key: string]: string };
-  // Label to dash style.
-  dashes: { [key: string]: string };
-  title: { [key: string]: string };
+interface Style {
+  color: string;
+  dash: string;
 }
 
 /**
- * Return color and dash style given place names and stats names.
+ * Return color and dash style given place names and stats var dcids.
  */
 function computePlotParams(
   placeNames: string[],
-  statsNames: string[]
-): PlotParams {
-  const colors = {};
-  const dashes = {};
-  const title = {};
-  const colorFn = getColorFn(statsNames);
-  const dashFn = getDashes(placeNames.length);
-  for (const statsName of statsNames) {
-    colors[statsName] = colorFn(statsName);
-  }
-  for (let i = 0; i < placeNames.length; i++) {
-    dashes[placeNames[i]] = dashFn[i];
+  statsVars: string[]
+): { [key: string]: Style } {
+  const defaultColor = getColorFn([""])("a");
+  const defaultDash = "";
+  const result = {};
+  if (statsVars.length === 1) {
+    // One stats var, colors are based on places and dash is solid.
+    const colorFn = getColorFn(placeNames);
+    result[statsVars[0]] = { color: defaultColor };
+    for (const placeName of placeNames) {
+      result[placeName] = { color: colorFn(placeName), dash: defaultDash };
+    }
+  } else if (placeNames.length === 1) {
+    // One place, colors are based on stats var and dash is solid.
+    const colorFn = getColorFn(statsVars);
+    result[placeNames[0]] = { color: "#000", dash: defaultDash };
```

Comment:
why is the default color black? can we make it a const?

Reply:
Made a constant